contracts/Strategy.sol

```diff
@@ -6,7 +6,10 @@ pragma solidity 0.6.12;
 pragma experimental ABIEncoderV2;
 
 // These are the core Yearn libraries
-import {BaseStrategy, StrategyParams} from "@yearnvaults/contracts/BaseStrategy.sol";
+import {
+    BaseStrategy,
+    StrategyParams
+} from "@yearnvaults/contracts/BaseStrategy.sol";
```

Comment:
Is this a new linter?

Reply:
Good question, i thought it was applying it from the project but i guess is using my global config which is similar to the configs from vaults v2 repo, should i undo the style changes or is that good to push?